app/build.gradle

```diff
@@ -52,12 +52,15 @@ dependencies {
     implementation composeBom
     androidTestImplementation composeBom
 
-    implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
-    implementation 'androidx.activity:activity-compose:1.6.1'
     implementation 'androidx.compose.ui:ui'
     implementation 'androidx.compose.ui:ui-tooling-preview'
     implementation 'androidx.compose.material3:material3'
+    implementation 'androidx.compose.material:material-icons-extended'
```

Comment:
If I remember correctly it is not recommended to add it because it is very heavy in terms of size :D It increases APK size a lot, but probably we can think about it later.

Reply:
You're right. It is said at the maven repository https://mvnrepository.com/artifact/androidx.compose.material/material-icons-extended. If we're only going to use a few of them I think a better way would be to add drawables to our resources for only those icons that we need.